plugins/autogems.cpp

```diff
@@ -146,6 +190,9 @@ void create_jobs() {
             continue;
         }
 
+        if (workshop->isForbidden())
+            continue;
+
```

Comment:
I'd have to check the original. I think the check was originally done somewhere else.